discovery/seata-discovery-namingserver/src/test/java/org/apache/seata/discovery/registry/namingserver/NamingserverRegistryServiceImplTest.java

```diff
@@ -157,6 +203,7 @@ public void testRegister2() throws Exception {
 
 
     @Test
+    @Disabled
```

Comment:
why there are some many unit test disabled?

Reply:
Because these test cases need to be coordinated with the namingserver node, they can only be used for local testing. The test cases in this test class were all initially disabled. I have modified and enabled some of them to run in GitHub CI.